panel/src/components/Navigation/Button.vue

```diff
@@ -307,7 +307,6 @@ export default {
 	--button-color-text: var(--button-color-dimmed-off);
 }
 .k-button:where([data-theme][data-variant="dimmed"]) {
-	--button-color-icon: var(--theme-color-icon);
```

Comment:
Removing this line breaks the colors for the dimmed buttons. 
![Screenshot 2024-04-23 at 12 19 00](https://github.com/getkirby/kirby/assets/24532/9802d578-08e0-4419-91a0-b2c1830d6a44)

Reply:
Ahh got it, we can out it back in.